### PR TITLE
Moving funding_statuses from Constants.yml along with the References [#152723649]

### PR DESCRIPTION
**Pull request by ying-pbrc**
https://www.pivotaltracker.com/n/projects/1918597/stories/152723649

### CODE REVIEW

**Comment by ying-pbrc** [comment]
I worked on this story based on v3.1.0 branch (a429ef5a78c9039ee466d4cfe2e31a931cec1d9d). I fixed a few minor travis issues, but there are still a lot of  failures in the latest build, https://travis-ci.org/sparc-request/sparc-request/jobs/309017114, particularly in spec/features/dashboard/protocols/create and spec/features/dashboard/protocols/edit. Please take a look when you get a chance. Thanks!

**Comment by ying-pbrc** [comment]
rspec spec/features/* failures were due to the missing funding_statuses in spec/support/permissible_values.rb. It's fixed.

File: app/views/protocols/view_details/_project_fields.html.haml [hunk 1]
```diff
@@ -39,7 +39,7 @@
       %label.col-lg-4
         = t(:protocols)[:projects][:information][:funding_status]
       .col-lg-8
-        = protocol.funding_status
+        = protocol.funding_status.nil? ? '' : protocol.funding_status.titleize
```
Comment (kyle-glick) [inline app/views/protocols/view_details/_project_fields.html.haml]: `protocol.funding_status.try(&:titleize)` should do the same thing but a little simpler

**Comment by ying-pbrc** [comment]
@kyle-glick, updated code as suggested.

**Pull request merged**